src/sentry/tagstore/snuba/backend.py

```diff
@@ -387,11 +387,15 @@ def get_group_event_filter(self, project_id, group_id, environment_id, tags):
             'environment': [environment_id],
             'issue': [group_id],
         }
+        aggregations = [
+            ['max', SEEN_COLUMN, 'last_seen'],
+        ]
```

Comment:
This is kind of weird and counterintuitive since this is being aggregated by `event_id` and not `issue`. This should(?) only be aggregating a single column for each "aggregate" but needs to happen to ensure that the `ORDER BY` clause is under an aggregate function.

Reply:
Is the groupby even necessary anymore? Its no longer required by the schema, and there is now another arg `selected_columns` that can be used to add extra columns that are not in groupings or aggregates.

Reply:
It might not be — I'll take a look. I didn't really think about it and assumed that this was maybe a side effect of the deletions stuff, but that doesn't really make any sense (and this would probably be the wrong place to handle it anyway.)